nuxeo-features/nuxeo-platform-mail/nuxeo-platform-mail-web/src/main/resources/OSGI-INF/nxmail-layouts-contrib.xml

```diff
@@ -369,7 +369,7 @@
           <property name="template">/widgets/email_text_widget_template.xhtml</property>
         </properties>
         <fields>
-          <field>#{document.id}</field>
+          <field>#{currentDocument.id}</field>
         </fields>
```

Comment:
I wonder if configuring this directly with <field>id</field> would be enough.

The "document" variable has indeed been removed from the context (alias to the "value" variable), here if the value matches the layout value, no need to hardcode the currentDocument variable: the layout field binding logics will build an equivalent expression.